sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3826,7 +3826,8 @@ object functions {
   }
 
   /**
-   * Creates timestamp from the number of seconds since UTC epoch.
+   * Converts the number of seconds from unix epoch (1970-01-01 00:00:00 UTC) 
```

Comment:
Linter complains about this

```suggestion
   * Converts the number of seconds from unix epoch (1970-01-01 00:00:00 UTC)
```